Assets/Dependencies/SmashBrew/Character/Character.cs

```diff
@@ -192,7 +190,23 @@ private enum FacingMode {
             return false;
         }
 
-        public float movementspeed = 0;
+        /// <summary>
+        /// How fast the character is currently walking/running.
+        /// </summary>
+        public float movementSpeed = 0;
```

Comment:
Do not use public variables. This is bad practice. If you want a variable serialized and want to expose it, use a private variable serialized using the SerializeField attribute and a public accessor property.

Runtime variables that shouldn't be serialized like isDashing should simply not have the SerializeField attribute to avoid serializing excess data.

Try to keep a consistent style with the rest of the project, Private variables should be "_camelCase". Public properties, variables, and methods should be capitalized.